apps/designer/app/designer/features/props-panel/props-panel.tsx

```diff
@@ -35,20 +42,34 @@ const Property = ({
   const type = argType?.control.type || "text";
   const defaultValue = argType?.control.defaultValue;
   const options = argType?.options;
+  const allProps = meta.argTypes ? Object.keys(meta.argTypes) : [];
+
   return (
     <Grid
       gap="1"
       css={{ gridTemplateColumns: "1fr 1fr auto", alignItems: "center" }}
     >
-      <TextField
-        readOnly={required}
-        variant="ghost"
-        placeholder="Property"
+      <Combobox
         name="prop"
+        items={allProps}
         value={prop}
-        autoFocus={prop === ""}
-        onChange={(event) => {
-          onChange(id, "prop", event.target.value);
+        itemToString={(item) => item ?? ""}
+        onItemSelect={(value) => {
+          onChange(id, "prop", value);
+        }}
+        disclosure={({ inputProps, toggleProps }) => (
+          <ComboboxTextField
+            toggleProps={toggleProps}
+            inputProps={{
+              ...inputProps,
+              readOnly: required,
+              placeholder: "Property",
+            }}
+          />
+        )}
+        contentProps={{
+          align: "start",
+          sideOffset: 5,
```

Comment:
Generally I consider this kind of props a red flag and a sign of the need for compound components. Why do you want to expose them?

Reply:
I felt like it looked weird when the tooltip is aligned horizontally in the middle relative to that input, it feels in the sidebar as if it should be in the middle of the sidebar, need to discuss this

Reply:
we could expose those props on the first level though, instead of contentProps, it was just simpler to forward them like this

Reply:
problem is that PopperContent is a component, if it was a hook, it would be simpler

Reply:
actually we can just export that PopperContent as a Content and let consumer pass those things directly to it

Reply:
yeah, let me do this right now

Reply:
done